third_party/ibis/ibis_teradata/datatypes.py

```diff
@@ -169,7 +169,14 @@ def to_ibis_from_SZ(cls, col_data, return_ibis_type=True):
                 timezone="UTC", nullable=cls._col_data_nullable(col_data)
             )
 
-        return "TIMESTAMP"
+        return "TIMESTAMP WITH TIME ZONE"
+    
+    @classmethod
+    def to_ibis_from_TZ(cls, col_data, return_ibis_type=True):
```

Comment:
Teradata type `TZ` is `TIME WITH TIME ZONE`, not `TIMESTAMP`. Is there a reason for this new function? 

The core data types test table doesn't have a time column because not all engines support it. I've never tested DVT with one, I wonder if we need a new test table. Not for this issue but for a future issue.

Reply:
Ah, good catch. I'll remove this - I thought it was another abbreviation for `TIMESTAMP WITH TIME ZONE`